python/pyarrow/parquet.py

```diff
@@ -541,6 +541,8 @@ def _sanitize_table(table, new_schema, flavor):
     The serialized Parquet data page format version to write, defaults to
     1.0. This does not impact the file schema logical types and Arrow to
     Parquet type casting behavior; for that use the "version" option.
+use_compliant_nested_type : bool, default False
+    Write compliant Parquet nested type, defaults to disable.
```

Comment:
Can you expand the documentation a bit? Eg a user will wonder: what does "compliant" mean in this specific case, and what is different with the default of False?

Reply:
I've expanded documentation, please let me know if you are happy about it. :)